test/blackbox-tests/test-cases/install-include/install-include-invalid-file.t

```diff
@@ -0,0 +1,31 @@
+Including a file in the install stanza which does not contain a sexp list
+
+  $ echo "(lang dune 3.5)" > dune-project
+
+  $ touch hello.opam
```

Comment:
If you include `(package (name hello))` in the dune-project file, you can skip this file.